Update refunds.md (#9687)
PostHog/posthog.com: contents/handbook/growth/sales/refunds.md

showTitle: true
---

We know things happen and sometimes you might need a refund. Here’s how we handle common scenarios:

### Learning Curve
Just got off of the startup plan/new client accidentally used us a lot.

We issue refunds in this category if this is the first bill >$1 and/or they meet criteria as explained above.

### Unexpected Stardom
Side project sudden volume spike

> We issue refunds in this category if this is the first bill >$1 and/or usage spiked by >200% compared to their average usage over the past 3 months, and/or the account is a non-commercial or a hobby project.

Reviewer: ```suggestion
We issue refunds in this category if this is the first bill >$1 and/or usage spiked by >200% compared to their average usage over the past 3 months, and the company doesn't have any revenue, or is a hobby project.
```

Reviewer: Tightened up slightly
